packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -176,6 +176,46 @@ export class IndexedDbPersistence implements Persistence {
    */
   static MAIN_DATABASE = 'main';
 
+  static async createIndexedDbPersistence(
+    persistenceKey: string,
+    clientId: ClientId,
+    platform: Platform,
+    queue: AsyncQueue,
+    serializer: JsonProtoSerializer
+  ): Promise<IndexedDbPersistence> {
+    const persistence = new IndexedDbPersistence(
+      persistenceKey,
+      clientId,
+      platform,
+      queue,
+      serializer
+    );
+    await persistence.start();
+    return persistence;
+  }
+
+  static async createMultiClientIndexedDbPersistence(
+    persistenceKey: string,
+    clientId: ClientId,
+    platform: Platform,
+    queue: AsyncQueue,
+    serializer: JsonProtoSerializer,
+    multiClientParams: {
+      sequenceNumberSyncer: SequenceNumberSyncer;
```

Comment:
Optional: You could create a type for this since you are using it in three (?) places now.

Reply:
Done.